libraries/stdlib/samples/test/samples/random/randoms.kt

```diff
@@ -38,4 +38,65 @@ class Randoms {
         // random with another seed produce another sequence
         assertPrints(randomValues3, "[14, 48, 57, 67, 82, 7, 61, 27, 14, 59]")
     }
-}
\ No newline at end of file
+
+    @Sample
+    fun nextBits() {
+        // always generates a 0
+        println(Random.nextBits(0))
+        // get a random 1 bit value
+        println(Random.nextBits(1))
+        // get a random 8 bit value
```

Comment:
These comments do not seem to be very informative, they don't clarify what the function does.
Consider writing something like "always generates 0 or 1" for `nextBits(1)`, and "generates a positive Int value less than 256" for `nextBits(8)`. 
`nextBits(16)` and `nextBits(32)` seem to be redundant as they do not add clarity, though `nextBits(32)` could be kept with comments alike "can generate any Int value"

Reply:
The comment for `Random.nextBits(0)` is just there because I thought it was an interesting bit of information.  I'll update the other comments to be more informative